pandas/tests/io/test_excel.py

```diff
@@ -2150,6 +2151,127 @@ def test_write_cells_merge_styled(self):
             self.assertEqual(xcell_b1.font, openpyxl_sty_merged)
             self.assertEqual(xcell_a2.font, openpyxl_sty_merged)
 
+    def test_styler_to_excel(self):
```

Comment:
I would put this test in another test class (in this file), so its a bit more natural. You can use fixtures /paramaterization if applicable